documentation/interpretation_guide.md

```diff
@@ -73,24 +83,32 @@ If there are restrictions for the circulation of private cars based on certain c
 - Formal sector workers are people who are employed with contracts, and pay taxes. Informal workers may be roadside vendors, work on markets, and do not have a formal contract for their work. 
 - If benefits in kind are being given (e.g. dry rations), this is recorded as a 0 with a note. It is not recorded as a non-zero value as it is not money being given or salaries replaced.
 - Edge cases and marginal examples
     - *Widening eligibility for unemployment benefit:* For example, if a nation or state had worked within the existing unemployment scheme to offer expanded eligibility during the COVID-affected period, for example giving benefits to those who wouldn’t normally be paid leave. This expands unemployment benefits to a wider group , and would therefore be recorded as the appropriate non-zero value depending on the value/coverage.
     - *Expedited access to unemployment benefit:* If a state takes policy action to make it faster and easier to access nation or  state/federal unemployment benefits during the COVID-affected period, but doesn’t expand eligibility criteria, this is only recorded as a note. The underlying policy hasn’t changed or expanded to reach more people.
 
+### E2- Debt/contract relief for households
+- This indicator does not have a binary flag variable to reflect geographic variation in policies. This implies that this policy applies generally across the whole jurisdiction. Therefore, we generally code the level of policy that applies everywhere across the jurisdiction- the highest common value of policy nationwide or statewide. We do not code any policies of a higher value that are only available in limited areas.  *For example*  if debt relief is only being provided by some cities, and not across the whole country or state, this cannot be coded. It is only recorded when the support recorded applies nationally or statewide.
 
 ### E3- Fiscal Measures
 
 - If the specific monetary value is undisclosed or unclear, we make a note recording the announcement, but don’t record the monetary value. 
 Sometimes governments make ambiguous funding announcements (where it is not clear what the money is for), or they announce multiple programmes under a single number (eg, fiscal stimulus, plus hospital funding, plus vaccine investment, plus support for other countries, in one number). In these cases, the spending is recorded in this E3 category, and not in the other monetary categories (E4, H4, and H5).
+- If the state is providing support to specific groups, for example funding children’s school meals, handing out staple goods, or providing grants to single parents, this should be coded as a note only for E3. As these forms of support are related to social welfare, they do not fit in E1 which covers income support and a replacement of lost salaries. E1 should be mostly about income support and income replacement, not general social programs, so it is more suitable to record such measures in a note for E3.
```

Comment:
I am not sure about this. My thinking is that you could record it as spending under E3... but you couldn't record it as income support under E1.